src/components/Suggestions.jsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import { Link } from "react-router-dom";
+
+const Suggestions = ({ suggestions = [] }) => (
+  <div>
+    <h3>Did you mean?</h3>
+    <ul>
+      {suggestions.map((text, magicKey) => (
```

Comment:
I think this is wrong and needs to destructure.

Reply:
i was wrong for now, but this has been updated to use the text instead of index.